library/idr/connections.py

```diff
@@ -0,0 +1,65 @@
+"""
+Helper functions for accessing the IDR from within IPython notebooks.
+"""
+import requests
+    
+def connection(host=None, user=None, password=None, port=4064):
+    """
+    Connect to the IDR analysis OMERO server
+    :return: A BlitzGateway object
+    """
+
+    # WARNING: The following block is automatically updated by the IDR
+    # deployment scripts. Do not edit this file without testing.
+# BEGIN ANSIBLE MANAGED BLOCK
+    HOSTNAME = "localhost"
```

Comment:
As mentioned elsewhere this will need to be synchronised and tested with a change to https://github.com/IDR/deployment/blob/IDR-0.4.1/ansible/idr-docker.yml#L184